src/test/shared/clients/defaultEc2Client.test.ts

```diff
@@ -93,6 +93,97 @@ describe('extractInstancesFromReservations', function () {
                 actualResult
             )
         })
+
+    it('is able to process results without complete Tag field.', async function () {
+        const testReservationsList: EC2.ReservationList = [
+            {
+                Instances: [
+                    {
+                        InstanceId: 'id1',
+                        Tags: [{ Key: 'Name', Value: 'name1' }],
+                    },
+                    {
+                        InstanceId: 'id2',
+                    },
+                ],
+            },
+            {
+                Instances: [
+                    {
+                        InstanceId: 'id3',
+                        Tags: [{ Key: 'Name', Value: 'name3' }],
+                    },
+                    {
+                        InstanceId: 'id4',
+                        Tags: [],
+                    },
+                ],
+            },
+        ]
+
+        const actualResult = await client.getInstancesFromReservations(intoCollection([testReservationsList])).promise()
+
+        assert.deepStrictEqual(
+            [
+                { InstanceId: 'id1', name: 'name1', Tags: [{ Key: 'Name', Value: 'name1' }] },
+                { InstanceId: 'id2' },
+                { InstanceId: 'id3', name: 'name3', Tags: [{ Key: 'Name', Value: 'name3' }] },
+                { InstanceId: 'id4', Tags: [] },
+            ],
+            actualResult
+        )
+    })
+})
+
+describe('getSingleInstanceFilter', function () {
+    const client = new Ec2Client('')
+
+    it('returns proper filter when given instanceId', function () {
+        const testInstanceId1 = 'test'
```

Comment:
is this test duplicate of line 189?

Reply:
Yep, good catch. Sometimes when I merge master back in I could weird stuff like this, but I missed this one.